addons/sale/migrations/12.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_renames = {
+    'payment_transaction': [
+        ('sale_order_id', None),
```

Comment:
here you are trying to rename the sale_order_id column. However, this column will only exist if you installed the sale_payment module on v11 https://github.com/odoo/odoo/blob/11.0/addons/sale_payment/models/payment.py#L16
Thus this migration you happen in the sale_payment migration module and not in the sale module.
For instance I don't have this sale_payment module installed and then I get this error:
```
2019-02-11 15:08:59,154 126 INFO openupgrade-to-12 OpenUpgrade: sale: pre-migration script called with version 11.0.1.1
2019-02-11 15:08:59,155 126 INFO openupgrade-to-12 OpenUpgrade: table payment_transaction, column sale_order_id: renaming to openupgrade_legacy_12_0_sale_order_id
2019-02-11 15:08:59,155 126 ERROR openupgrade-to-12 odoo.sql_db: bad query: b'ALTER TABLE "payment_transaction" RENAME "sale_order_id" TO "openupgrade_legacy_12_0_sale_order_id"'
ERROR: column "sale_order_id" does not exist                                                    
                                                                                                
2019-02-11 15:08:59,155 126 ERROR openupgrade-to-12 OpenUpgrade: sale: error in migration script sale/migrations/12.0.1.1/pre-migration.py: column "sale_order_id" does not exist
                                                                                                
2019-02-11 15:08:59,155 126 ERROR openupgrade-to-12 OpenUpgrade: column "sale_order_id" does not exist
                                                                                                
Traceback (most recent call last):                                                              
  File "/src/openupgradelib/openupgradelib/openupgrade.py", line 1445, in wrapped_function      
    if use_env2 else cr, version)                                                               
  File "/odoo/src/addons/sale/migrations/12.0.1.1/pre-migration.py", line 44, in migrate        
    openupgrade.rename_columns(env.cr, _column_renames)                                         
  File "/src/openupgradelib/openupgradelib/openupgrade.py", line 387, in rename_columns         
    'ALTER TABLE "%s" RENAME "%s" TO "%s"' % (table, old, new,))
  File "/odoo/src/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/odoo/src/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
psycopg2.ProgrammingError: column "sale_order_id" does not exist
```

Reply:
and the same happens for the next payment_tx_id column rename in the sale_order table!